frontend/src/views/patientregistration/ViewPatientRegistration.vue

```diff
@@ -176,7 +204,6 @@ export default {
     },
     resetForm() {
       this.phn = ''
-      this.payee = ''
```

Comment:
Why is this removed, payee field should be reset also?

Reply:
It will be a read only field once https://dev.azure.com/cgi-vic-hlth/HNWeb/_workitems/edit/7102 is done.